### PR TITLE
remove: `amdgpu-dkms-firmware-deb`

### PR DESCRIPTION
**Pull request by cat-master21**
Remove amdgpu-dkms-firmware-deb as a thought amdgpu-dkms was a dependency for aomp-deb. The issue I had however was not that but an issue with aomp itself. The CI may fail to install as https://raw.githubusercontent.com/pacstall/pacstall-programs/master/packages/aomp-deb/fix_AOMP.patch is not merged yet.

### CODE REVIEW

File: packages/aomp-deb/aomp-deb.pacscript [hunk 1]
```diff
@@ -1,8 +1,15 @@
 name="aomp-deb"
 gives="aomp"
 version="14.0-2"
+depends="libtinfo5"
+patch="https://raw.githubusercontent.com/pacstall/pacstall-programs/master/packages/aomp-deb/fix_AOMP.patch"
```
Comment (Henryws) [inline packages/aomp-deb/aomp-deb.pacscript:5]: Patch is an array

**Review by Henryws — CHANGES_REQUESTED** [review]

**Comment by cat-master21** [comment]
Should be ready. Note that pre-commit.ci should fail as it messes with my patch file.

File: packages/aomp-deb/aomp-deb.pacscript [hunk 2]
```diff
@@ -11,5 +11,5 @@ maintainer="cat-master21 <96554164+cat-master21@users.noreply.github.com>"
 
 postinst() {
 # patch to fix AOMP variable.
-sudo patch "/usr/lib/aomp/bin/mymcpu" < "./PACSTALL_patchesdir/fix_AOMP.patch"
+patch | sudo tee "/usr/lib/aomp/bin/mymcpu" < "./PACSTALL_patchesdir/fix_AOMP.patch"
```
Comment (Henryws) [inline packages/aomp-deb/aomp-deb.pacscript:14]:
```suggestion
patch | sudo tee "/usr/lib/aomp/bin/mymcpu" < "./PACSTALL_patchesdir/fix_AOMP.patch" >/dev/null
```

**Review by Henryws — CHANGES_REQUESTED** [review]
Redirect the tee output to `/dev/null` to prevent terminal output

**Comment by cat-master21** [comment]
Done @Henryws

File: packages/aomp-deb/aomp-deb.pacscript [hunk 3]
```diff
@@ -1,8 +1,15 @@
...
+patch=("https://raw.githubusercontent.com/pacstall/pacstall-programs/master/packages/aomp-deb/fix_AOMP.patch")
 breaks="${gives} ${gives}-git ${gives}-bin ${gives}-app"
 url="https://github.com/ROCm-Developer-Tools/aomp/releases/download/rel_${version}/aomp_Ubuntu1804_${version}_amd64.deb"
 description="Aomp is AMD's version of compiler Clang."
 hash="34abb49981623f7009ce7e06cc7d6aa935c4fbad67e2876e5b58ef72063a63b8"
 maintainer="cat-master21 <96554164+cat-master21@users.noreply.github.com>"
+
+postinst() {
+# patch to fix AOMP variable.
+patch | sudo tee "/usr/lib/aomp/bin/mymcpu" < "./PACSTALL_patchesdir/fix_AOMP.patch" > /dev/null
```
Comment (Henryws) [inline packages/aomp-deb/aomp-deb.pacscript:14]:
```suggestion
sudo patch | tee "/usr/lib/aomp/bin/mymcpu" < "./PACSTALL_patchesdir/fix_AOMP.patch" > /dev/null
```

**Review by Henryws — CHANGES_REQUESTED** [review]

**Comment by wizard-28** [comment]
@Henryws continue your review.

File: packages/aomp-deb/aomp-deb.pacscript [hunk 4]
```diff
@@ -1,8 +1,15 @@
...
+patch=("https://raw.githubusercontent.com/pacstall/pacstall-programs/master/packages/aomp-deb/fix_AOMP.patch")
 breaks="${gives} ${gives}-git ${gives}-bin ${gives}-app"
 url="https://github.com/ROCm-Developer-Tools/aomp/releases/download/rel_${version}/aomp_Ubuntu1804_${version}_amd64.deb"
 description="Aomp is AMD's version of compiler Clang."
 hash="34abb49981623f7009ce7e06cc7d6aa935c4fbad67e2876e5b58ef72063a63b8"
 maintainer="cat-master21 <96554164+cat-master21@users.noreply.github.com>"
+
+postinst() {
+# patch to fix AOMP variable.
+sudo patch | tee "/usr/lib/aomp/bin/mymcpu" < "./PACSTALL_patchesdir/fix_AOMP.patch" > /dev/null
```
Comment (Henryws) [inline packages/aomp-deb/aomp-deb.pacscript:14]:
```suggestion
	# patch to fix AOMP variable.
	sudo patch | tee "/usr/lib/aomp/bin/mymcpu" < "./PACSTALL_patchesdir/fix_AOMP.patch" > /dev/null
```

**Review by Henryws — CHANGES_REQUESTED** [review]
Styling issues

**Review by Henryws — CHANGES_REQUESTED** [review]
https://github.com/pacstall/pacstall-programs/runs/5297309951?check_suite_focus=true#step:9:79

**Comment by cat-master21** [comment]
This may be because the file: https://raw.githubusercontent.com/pacstall/pacstall-programs/master/packages/aomp-deb/fix_AOMP.patch does not exist.

**Review by Henryws — APPROVED** [review]

**Pull request merged**